testdata/recover.go

```diff
@@ -22,6 +27,9 @@ func main() {
 
 	println("\n# defer panic")
 	deferPanic()
+
+	println("\n# runtime.Goexit")
```

Comment:
Shouldn't `recover.txt` golden output also be updated here?

Reply:
Oops, yes! Fixed now :grimacing: